### ISSUE TITLE
Build failure on osx

### ISSUE THREAD

**Issue by errm** [projectatomic/skopeo#380]
As per the README

```
brew install gpgme
git clone https://github.com/projectatomic/skopeo $GOPATH/src/github.com/projectatomic/skopeo
cd $GOPATH/src/github.com/projectatomic/skopeo && make binary-local
```

Output:

```
go build -ldflags "-X main.gitCommit=91606d49f2f7d17c226940e98b5f42500a4c3efb" -gcflags "" -tags "btrfs_noversion libdm_no_deferred_remove" -o skopeo ./cmd/skopeo
# pkg-config --cflags ostree-1 ostree-1 ostree-1 ostree-1 ostree-1 ostree-1
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
Package ostree-1 was not found in the pkg-config search path.
Perhaps you should add the directory containing `ostree-1.pc'
to the PKG_CONFIG_PATH environment variable
No package 'ostree-1' found
pkg-config: exit status 1
make: *** [binary-local] Error 2
```

**Comment by rhatdan** [comment]
Looks like we need  a way to compile out the ostree support in containers/image for platforms that do not support it.

**Comment by mtrmac** [comment]
Thanks for your report.

Could you test and report back whether
```sh
make binary-local BUILDTAGS=containers_image_ostree_stub
```
works, please? If so, we can at least update the README, or a tested PR automating this for macOS would be also welcome.

(I can’t find a brew recipe for libostree, and with https://github.com/ostreedev/ostree/blob/master/src/libostree/ostree-linuxfsutil.c it seems unlikely that it would easily build for macOS, so compiling it out seems to be the right thing to do.)

**Comment by rhatdan** [comment]
Should we just add an ostree tag, and then only build ostree if this is specified?  Then we can get all of our Linux packages to specify ostree?

**Comment by errm** [comment]
```
make binary-local BUILDTAGS=containers_image_ostree_stub
```

does indeed work

**Comment by mtrmac** [comment]
Great, thanks for testing. #381 is a minimal README update; autodetecting macOS in `Makefile` and adding the tag to `LOCAL_BUILD_TAGS` would be nice but I’ll leave that up to someone with a macOS development environment set up.

**Comment by rhatdan** [comment]
LGTM (Although I hate the buildtag...

**Comment by mtrmac** [comment]
Is this fixed by #382?
